File: datas/templates/C++/Qt Form - Single Inheritance/$Class Name$.cpp
#include "$Class Name$.h"
#include "ui_$Class Name$.h"

#include <QEvent>

$Class Name$::$Class Name$( QWidget* parent )
    : $Qt Widget Class$( parent ), ui( new Ui_$Class Name$ )
{
    ui->setupUi( this );
}

$Class Name$::~$Class Name$()
{
    delete ui;
}

void $Class Name$::retranslateUi()
{
    ui->retranslateUi( this );
    // do your custom retranslate here
}

void $Class Name$::changeEvent( QEvent* event )
{
    $Qt Widget Class$::changeEvent( event );
    
    if ( event->type() == QEvent::LanguageChange ) {
        retranslateUi();
    }
}

File: datas/templates/Python/Qt Form - Single Inheritance/$Class Name$.py
from PyQt4 import uic

( Ui_$Class Name$, $Qt Widget Class$ ) = uic.loadUiType( '$Class Name$.ui' )

class $Class Name$ ( $Qt Widget Class$ ):
    """$Class Name$ inherits $Qt Widget Class$"""

    def __init__ ( self, parent = None ):
        $Qt Widget Class$.__init__( self, parent )
        self.ui = Ui_$Class Name$()
        self.ui.setupUi( self )

    def __del__ ( self ):
        self.ui = None
